### DISCUSSION TITLE
Adding poems

### DISCUSSION THREAD

**Discussion by OfficialPhilcomm** [lukePeavey/quotable#112] [Ideas]
I was recently searching for more recent quotes, and I realized that a lot of longer quotes are more like poems. So I thought this could be a new section for the api. What do you think @lukePeavey? Is that something you'd be interested in adding?

If you are interested but don't have the time, I will try to set up the project locally. My JS is a bit rusty, but I'll probably be able to add it too.

**Comment by lukePeavey** [comment]
Could you give me an example of the kind of thing you would like to add?

**Reply by OfficialPhilcomm** [reply]
```
Did you hear about the rose that grew
from a crack in the concrete?
Proving nature's law is wrong it
learned to walk with out having feet.
Funny it seems, but by keeping its dreams,
it learned to breathe fresh air.
Long live the rose that grew from concrete
when no one else ever cared.
```
The main difference is that these are usually longer and contain 
linebreaks.
I don't think it would be a good idea to merge these into the quotes endpoint, because of their different formatting, but rather have a different endpoint like `/poems`.

**Comment by lukePeavey** [comment]
let me think on this... on will get back you soon
